Content.Client/UserInterface/Controls/RadialMenu.cs

```diff
@@ -229,6 +252,69 @@ public RadialMenuTextureButton()
         OnButtonUp += OnClicked;
     }
 
+    /// <inheritdoc />
+    protected override void Draw(DrawingHandleScreen handle)
+    {
+        var texture = TextureNormal;
+
+        if (texture == null)
+        {
+            TryGetStyleProperty(StylePropertyTexture, out texture);
+            if (texture == null)
+            {
+                return;
+            }
+        }
+        // draw texture
+        handle.DrawTextureRectRegion(texture, PixelSizeBox);
+
+        // draw sector where space that button occupies actually is
+        var pX = -Position.X + Parent!.Width / 2;
+        var pY = -Position.Y + Parent.Width / 2;
+
+        var containerCenter = new Vector2(pX, pY) * UIScale;
+
+        const float singleSegmentSize = MathF.Tau / 32;
+
+        var controlSegmentSize = AngleSectorTo - AngleSectorFrom;
+        var segCount = (int)(controlSegmentSize / singleSegmentSize) + 1;
+
+        // CHANGE TO STACKALLOC AND MOVE THIS STUFF TO TOOLBOX!111
+        var buffer = new Vector2[segCount * 2];
+
+        if (Parent is not RadialContainer container)
+        {
+            return;
+        }
+
+        var radius = container.Radius * UIScale;
+
+        for (var i = 0; i < segCount; i++)
+        {
+            float angle;
+            if (i == segCount - 1)
+            {
+                // fix rounding problem that was created when calculating count of segments as int
+                angle = AngleSectorTo;
+            }
+            else
+            {
+                angle = AngleSectorFrom + singleSegmentSize * i;
+            }
+
+            var point = new Angle(angle).RotateVec(-Vector2.UnitY);
+
+            buffer[i * 2] = containerCenter + point * radius * 2;
+            buffer[i * 2 + 1] = containerCenter + point * radius / 2;
+        }
+
+        var color = DrawMode == DrawModeEnum.Hover
```

Comment:
same as stated before about stylesheets but with 1 more problem - this is TextureButton but i need styles for ContainerButton coz i think only it have background colors that i can use. Should i create separate bunch of code in stylesheets just for that? It feels wierd :(

Reply:
If you want them, then yes

Reply:
Maybe too lofty for this PR, but it sure would be nice to be able to style the segments using something similar to a `StyleBox` - I'd really like to see a `StyleBoxTexture` used to apply more than just a solid colour to the segment.

Reply:
i'll leave Color properties for now, i think i spent way too much with this PR - styling in the way i was hoping for is not possible due to sytlesheet mechanism limitations, and adding new style elements to all themes is not what i want in this pr - it is already starting to bloat! We need to merge it before it is dead.